modules/tinymce/src/core/test/ts/browser/html/DomParserTest.ts

```diff
@@ -1448,6 +1448,43 @@ describe('browser.tinymce.core.html.DomParserTest', () => {
 
         assert.equal(serializedHtml, expectedHtml);
       });
+
+      const testSplitInvalidBlocksOut = (testCase: { input: string; expected: string }) => {
```

Comment:
Should this function be moved to the top of the context-block or top of the description? Additionally, should it be named 'assert' instead of 'test'?